discovery-infra/update_bm_inventory_cm.py

```diff
@@ -21,7 +21,7 @@
     ("INVENTORY_PORT", ""),
     ("AGENT_DOCKER_IMAGE", ""),
     ("KUBECONFIG_GENERATE_IMAGE", ""),
-    ("BASE_DNS_DOMAINS", ":"),
+    ("BASE_DNS_DOMAINS"),
```

Comment:
u must have have 2 params here, it will try to evn_data = os.getenv(env[0], env[1])
and fail if it is not set

Reply:
It actually doesn't fail as it treats env as a string :) but pushed a new patch with a proper fix.